src/Servers/Kestrel/Core/test/HttpParserTests.cs

```diff
@@ -90,11 +90,15 @@ public void ParseRequestLineThrowsOnInvalidRequestLine(string requestLine)
             var buffer = new ReadOnlySequence<byte>(Encoding.ASCII.GetBytes(requestLine));
             var requestHandler = new RequestHandler();
 
+#pragma warning disable CS0618 // Type or member is obsolete
             var exception = Assert.Throws<BadHttpRequestException>(() =>
```

Comment:
This is a good example of where we want to keep the pragmas so we can test that we're throwing the old exception type.